code/game/objects/structures/platforms.dm

```diff
@@ -79,6 +93,49 @@
 			SPAN_WARNING("You finish dragging \the [user.pulling] onto \the [src]."))
 			user.pulling.forceMove(move_to_turf)
 
+/obj/structure/platform/attack_alien(mob/living/carbon/xenomorph/user)
+	if(stat & BROKEN)
+		to_chat(user, SPAN_WARNING("Its already destroyed!"))
+		return XENO_NO_DELAY_ACTION
+
+	if(stat & explo_proof)
+		to_chat(user, SPAN_WARNING("Its too strong for us!"))
+		return XENO_NO_DELAY_ACTION
+
+	if(user.action_busy)
+		return XENO_NO_DELAY_ACTION
+	if(user.a_intent != INTENT_DISARM)
```

Comment:
Move the `action_busy` and `user.a_intent != INTENT_DISARM` check before the `BROKEN` and `explo_proof` checks because otherwise you get error messages when you aren't even in disarm mode clicking on platforms.